diff -urN wine-tkg-scripts/wine64-tkg b/wine64-tkg
--- wine-tkg-scripts/wine64-tkg	2025-08-25 23:43:14.913169286 +0800
+++ b/wine64-tkg	2025-08-28 15:05:03.918804765 +0800
@@ -19,6 +19,8 @@
   export WINEDLLPATH="/usr/lib/x86_64-linux-gnu/wine:/usr/lib/i386-linux-gnu/wine"
 fi
 
+echo -e "\e[36m"
+
 cat >&2 <<'EOM'
        .---.`               `.---.
     `/syhhhyso-           -osyhhhys/`
@@ -41,6 +43,10 @@
 
 EOM
 
+echo -e "\e[1m [wine-tkg-builds patch] \e[0m (\e[4mhttps://github.com/Waim908/wine-tkg-builds\e[0m)"
+
+echo -e "\n\n\n ᗜˬᗜ"
+
 if [ -x "$(command -v _wine64)" ]; then
   _wine64 "${@:1}"
 else
diff -urN wine-tkg-scripts/wine-tkg b/wine-tkg
--- wine-tkg-scripts/wine-tkg	2025-08-25 23:43:14.913169286 +0800
+++ b/wine-tkg	2025-08-28 15:04:21.616893042 +0800
@@ -19,6 +19,8 @@
   export WINEDLLPATH="/usr/lib/x86_64-linux-gnu/wine:/usr/lib/i386-linux-gnu/wine"
 fi
 
+echo -e "\e[36m"
+
 cat >&2 <<'EOM'
        .---.`               `.---.
     `/syhhhyso-           -osyhhhys/`
@@ -41,6 +43,10 @@
 
 EOM
 
+echo -e "\e[1m [wine-tkg-builds patch] \e[0m (\e[4mhttps://github.com/Waim908/wine-tkg-builds\e[0m)"
+
+echo -e "\n\n\n ᗜˬᗜ"
+
 if [ -x "$(command -v _wine)" ]; then
   _wine "${@:1}"
 else
diff -urN wine-tkg-scripts/wine-tkg-interactive b/wine-tkg-interactive
--- wine-tkg-scripts/wine-tkg-interactive	2025-08-25 23:43:14.913169286 +0800
+++ b/wine-tkg-interactive	2025-08-28 15:07:28.457503130 +0800
@@ -19,6 +19,8 @@
   export WINEDLLPATH="/usr/lib/x86_64-linux-gnu/wine:/usr/lib/i386-linux-gnu/wine"
 fi
 
+echo -e "\e[36m"
+
 cat >&2 <<'EOM'
        .---.`               `.---.
     `/syhhhyso-           -osyhhhys/`
@@ -41,6 +43,10 @@
 
 EOM
 
+echo -e "\e[1m [wine-tkg-builds patch] \e[0m (\e[4mhttps://github.com/Waim908/wine-tkg-builds\e[0m)"
+
+echo -e "\n\n\n ᗜˬᗜ"
+
 read -rp "Please enter your wine command ( examples: wine \"/path/to/your/app\" ; winecfg ; regedit ): " _wine_tkg_command
 
 $_wine_tkg_command
